internal/driver/driver.go

```diff
@@ -73,6 +79,47 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *sdkModel.As
 	return nil
 }
 
+// creates the options to connect with a opcua Client based on the configured options.
+func (d *Driver) createClientOptions() []opcua.Option {
+	availableServerEndpoints, err := opcua.GetEndpoints(d.serviceConfig.OPCUAServer.Endpoint)
+	if err != nil {
+		d.Logger.Error("OPC GetEndpoints: %w", err)
+	}
+	credentials, err := getCredentials(d.serviceConfig.OPCUAServer.CredentialsPath)
+	if err != nil {
+		d.Logger.Error("getCredentials: %w", err)
```

Comment:
Found also this place where error is more or less ignored. But members of `credentials` are accessed below!

Similar applies to the `availableServerEndpoints` above which are also further used below in case of a possible error.

Reply:
done